Lib/pathlib.py

```diff
@@ -1245,6 +1247,21 @@ def replace(self, target):
         self._accessor.replace(self, target)
         return self.__class__(target)
 
+    def move(self, target, copy_function=shutil.copy2):
+        """
+        Recursively move a file or directory to another location (target),
+        using ``shutil.move``.
+
+        If the destination is on the current filesystem, then os.rename() is used.
+        Otherwise, src is copied to dst using copy_function and then removed.
+        In case of symlinks, a new symlink pointing to the target of will be
+        created in or as target and the original path will be removed.
```

Comment:
Can you re-word this sentence? I don't follow it I'm afraid.

Reply:
See my rephrasing. Maybe it's better now.